.github/workflows/on-pr.yaml

```diff
@@ -12,6 +12,26 @@ jobs:
   test-and-build:
     name: Test & Build
     runs-on: ubuntu-latest
+    outputs:
+      package_version: ${{ steps.package_version.outputs.PACKAGE_VERSION }}
```

Comment:
should probably be only in build, not here.

Reply:
:+1: